test/pantry-testing-data.js

```diff
@@ -0,0 +1,81 @@
+const pantryTestingData = [
```

Comment:
Awesome job putting the test data in its own file

Reply:
Agreed with Riley here, appreciate the test data file for a much cleaner test file.